package.json

```diff
@@ -52,9 +52,10 @@
   },
   "dependencies": {
     "lodash.unescape": "4.0.0",
-    "object-assign": "^4.0.1"
+    "object-assign": "^4.0.1",
+    "semver": "^4.3.6"
```

Comment:
Can't think of one. I looked at the commit log and seems they only dropped support for AMD modules and minifying the code in v5. Do you want to create a PR?